src/core/types/chains.ts

```diff
@@ -20,6 +21,26 @@ export const chainHardhat: Chain = {
   testnet: true,
 };
 
+export const chainBlast: Chain = {
+  id: BLAST_CHAIN_ID,
+  name: 'Blast',
+  network: 'blast',
+  rpcUrls: {
+    public: { http: [process.env.BLAST_MAINNET_RPC as string] },
+    default: {
+      http: [process.env.BLAST_MAINNET_RPC as string],
+    },
+  },
+  blockExplorers: {
+    default: { name: 'Blastscan', url: 'https://blastscan.io/' },
+  },
+  nativeCurrency: {
+    name: 'Blast',
+    symbol: 'BLAST',
+    decimals: 18,
+  },
+};
```

Comment:
can't we update wagmi/viem? I think they got blast already

Reply:
i updated it & didn't see it

Reply:
Should be in `2.7.18` and above

Reply:
bumping

Reply:
the new Chain object from viem is different than Chain object from wagmi 🙃

Reply:
Wat

Reply:
https://github.com/rainbow-me/browser-extension/blob/master/package.json#L156

we're very late